src/core/client/admin/routes/Moderate/Queue/ForReviewQueueRoute/ForReviewQueueRoute.tsx

```diff
@@ -0,0 +1,292 @@
+          // eslint-disable-next-line no-console
+          console.error(error);
+        }
+      }
+    );
+  }, [relay]);
+
+  const onReview = useCallback(
+    async (id: string, reviewed: boolean) => {
+      await markFlagged({ id, reviewed });
+    },
+    [markFlagged]
+  );
+
+  if (!query || !query.viewer) {
+    return null;
+  }
+
+  if (isRefetching) {
+    return <LoadingQueue />;
+  }
+
+  const flagActions = query.flags.edges.map((edge: { node: any }) => edge.node);
+
+  return (
+    <IntersectionProvider>
+      <HorizontalGutter size="double">
+        <Table fullWidth className={styles.table}>
+          <TableHead>
+            <TableRow>
+              <TableCell>
+                <Localized id="forReview-time">Time</Localized>
+              </TableCell>
+              <TableCell>
+                <Localized id="forReview-comment">Comment</Localized>
+              </TableCell>
+              <TableCell>
+                <Localized id="forReview-reportedBy">Reported by</Localized>
+              </TableCell>
+              <TableCell>
+                <Localized id="forReview-reason">Reason</Localized>
+              </TableCell>
+              <TableCell>
+                <Localized id="forReview-description">Description</Localized>
+              </TableCell>
+              <TableCell>
+                <Localized id="forReview-reviewed">Reviewed</Localized>
+              </TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {flagActions.map((a, i) => (
+              <ForReviewQueueRow
+                key={a.id}
+                id={a.id}
+                comment={{ ...a.comment, revision: a.revision }}
+                username={a.flagger ? a.flagger.username : ""}
+                reason={a.reason}
+                additionalDetails={a.additionalDetails}
+                createdAt={a.createdAt}
+                reviewed={a.reviewed}
+                onReview={onReview}
+              />
+            ))}
+          </TableBody>
+        </Table>
+        {relay.isLoading() && (
+          <Flex justifyContent="center">
+            <Spinner />
+          </Flex>
+        )}
+        {relay.hasMore() && (
+          <Flex justifyContent="center">
+            <AutoLoadMore
+              disableLoadMore={disableLoadMore}
+              onLoadMore={loadMore}
+            />
+          </Flex>
+        )}
+      </HorizontalGutter>
+    </IntersectionProvider>
+  );
+};
+
+type FragmentVariables = ForReviewQueueRoutePaginationQueryVariables;
+
+const enhanced = withPaginationContainer<
+  Props,
+  ForReviewQueueRoutePaginationQueryVariables,
+  FragmentVariables
+>(
+  {
+    query: graphql`
+      fragment ForReviewQueueRoute_query on Query
+        @argumentDefinitions(
+          count: { type: "Int", defaultValue: 5 }
+          cursor: { type: "Cursor" }
+          storyID: { type: "ID" }
+          siteID: { type: "ID" }
+          section: { type: "SectionFilter" }
```

Comment:
Needs support for sites, stories, and sections.

Reply:
Otherwise I end up here when I use the filters:
<img width="898" alt="grafik" src="https://user-images.githubusercontent.com/14221600/119551067-7ecde200-bd99-11eb-8c91-997d6c681715.png">

Reply:
Yeah, I have a patch incoming

Reply:
This is patched and fixed now